src/sass/style.scss

```diff
@@ -15,10 +15,10 @@
 
 
 // ITCSS: FRAMEWORKS/THEMES
-@import "../../node_modules/foundation-sites/dist/css/foundation.min.css";
+@import "../../node_modules/foundation-sites/dist/css/foundation.min";
```

Comment:
`import "file.css"` is valid CSS and when scene by the browser will trigger a network request for "file.css". The page and CSS is paused until this request completes, so it basically halts all rendering and code execution.

Since `import "file.css";` is valid CSS and Sass is a superset of it, then it does not process this line.

However if you leave off the file extension, it is no longer valid CSS, and then Sass's path resolving kicks in, looks for a `.sass`, `.scss`, or `.css` file with that name, then does the usual import of the file contents.

Don't feel bad, I did the same thing on the VGT import below